syncer.go

```diff
@@ -65,7 +64,7 @@ type Syncer struct {
 	metricsHandle          *sqmetrics.SquareMetrics
 	syncMutex              sync.Mutex
 	pollInterval           time.Duration
-	lastSuccessMonotonic   uint64
+	lastSuccess            uint64
```

Comment:
I think for monotonic time to work properly, we need to store the value of time.Now() here, rather than round-tripping through a uint64.

Reply:
good point!